ervinu/web content/style.css

```diff
@@ -1,105 +1,106 @@
-body {
-    background-color: rgb(211, 207, 207);
-    margin: 10px 5px;
-    font-family: sans-serif;
+a {
+    text-decoration: none;
 }
 
-#top-container {
-    background-color: #ffffff;
-    margin: 5px;
+.background-highlight {
+        background-color: #ff0000;
+        color: #ffffff;
+        display: inline-block;
+        margin-left: 4px;
 }
 
-#title h2, #title a {
-    display: inline-block;
-    margin: 5px 10px;
+body {
+    background-color: #d3cfcf;
+    margin: 8px 4px;
+    font-family: sans-serif;
 }
 
-a {
-text-decoration: none;
+.bottom-container {
+    margin-top: 16px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
 }
 
 .carousel {
-    background-color: rgb(255, 255, 255);
-    height: 330px;
-    width: 99%;
+    background-color: #ffffff;
+    height: 100%;
+    width: 100%;
     position: relative;
     display: flex;
+    justify-content: space-between;
 }
 
 .carousel p {
     margin: 4px 0px;
 }
 
 .carousel img {
-    background-color: rgb(236, 236, 236);
+    background-color: #ececec;
+    display: flex;
     height: 70%;
     width: 98%;
-    margin: 0px 10px;
+    margin: 0px 4px 0px 4px;
 }
 
-
-.deal-discount p:first-child {
-    background-color: rgb(255, 0, 0);
-    color: rgb(255, 255, 255);
-    display: inline-block;
-    margin-left: 10px;
-}
-
-.deal-discount p:last-child {
-    color: red;
-    display: inline-block;
-}
-
-.euro-sign {
-    font-size: 0.5em;
+.cents {
+    font-size: 8px;
     position: relative;
-    top: -0.25em;
-    right: -0.25em;
     top: -5px;
 }
 
-.cents {
-    font-size: 0.5em;
+.euro-sign {
+    font-size: 8px;
     position: relative;
     top: -5px;
-    right: -1px;
 }
 
-.was {
-    font-size: 0.8em;
-    opacity: 0.5;
-}
-
-.old-price {
-    font-size: 0.8em;
-    opacity: 0.5;
-    text-decoration: line-through;
+.galery {
+    background-color: #ffffff;
+    margin: 4px;
+    width: 330px;
+    overflow: hidden;
 }
 
-#bottom-container {
-    margin-top: 20px;
-    display: flex;
-    justify-content: space-between;
+.galery a {
+    margin: 8px 16px 8px;
 }
 
-.galery {
-    background-color: rgb(255, 255, 255);
-    margin: 5px;
-    width: 340px;
-    overflow: hidden;
+.galery h3 {
+    margin: 4px 16px;
 }
 
 .galery img {
-    margin: 0px 15px;
+    margin: 0px 16px;
     width: 90%;
     height: 80%;
     justify-content: space-between;
+    display: flex;
+    flex-direction: row;
 }
 
-.galery h3 {
-    margin: 5px 15px;
+.old-price {
+    font-size: 16px;
+    opacity: 0.5;
+    text-decoration: line-through;
 }
 
-.galery a {
-    margin: 10px 15px 10px;
+.top-container {
+    background-color: #ffffff;
+    margin: 4px;
+}
+
+.title h2, .title a {
+    display: inline-block;
+    margin: 4px 8px;
+}
+
+.text-highlight {
+    color: #ff0000;
+    display: inline-block;
+}
+
+.was {
```

Comment:
was is too vague. What means was? past?

Reply:
It was for the 'Was' word before the old price with the line-through.